docs/manifest.json

```diff
@@ -22,7 +22,8 @@
       "# Backend",
       ["Fastify", "/quickstarts/fastify"],
       "# Example Apps",
-      ["Next.js starter repo", "https://github.com/clerkinc/clerk-next-app"],
+      ["Next.js starter repo with App Router", "https://github.com/clerkinc/clerk-next-app"],
```

Comment:
Clerk-next-app is not a starter repo, it is the meatier example repo.

IMO, we can reference a repo in a quickstart, and remove this "Example apps" section altogether.
E.g. in the Next.js quickstart, something like "If you would like a repository with the basics to help you get started, check out the [starter repository](link). If you would like a repository that demo's many of the features Clerk has to offer, such as managing sessions and organizations, check out the [Clerk and Next.js showcase repository](link)."

We are still figuring out what we want to call these repository's (starter, or whatever), but this is an example of the copy we can have in quickstarts to guide users to the repositories.

Reply:
What about putting that recommendation at the end of the tutorial, rather than the beginning, as a possible next step?